pkg/lang/frontend/starlark/v0/config/config.go

```diff
@@ -45,9 +45,28 @@ var Module = &starlarkstruct.Module{
 		"rstudio_server": starlark.NewBuiltin(ruleRStudioServer, ruleFuncRStudioServer),
 		"entrypoint":     starlark.NewBuiltin(ruleEntrypoint, ruleFuncEntrypoint),
 		"repo":           starlark.NewBuiltin(ruleRepo, ruleFuncRepo),
+		"shm_size":       starlark.NewBuiltin(ruleShmSize, ruleFuncShmSize),
 	},
 }
 
+func ruleFuncShmSize(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var shmSize starlark.Int
+
+	if err := starlark.UnpackArgs(ruleShmSize, args, kwargs, "size?", &shmSize); err != nil {
```

Comment:
This argument should be required.

Reply:
Sorry. I don't understand what you mean, do I need to modify the starlark.Builtin parameter of the ruleFuncShmSize function to _?

Reply:
A function with only one optional parameter doesn't make sense. You should check the starlark go lib about the function interface.

Reply:
https://pkg.go.dev/go.starlark.net/starlark#UnpackArgs